Bottom: 94d6641a0a4f1abb240ba15e51c6318abc79a582
Top:    d47cf7e79908cd101d27676648d63c44cbc20fd2
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-08-28 03:12:15 +0100

Refresh of add-support-for-depth-to-trian

---

diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index 67784e0..60f9a3d 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -231,7 +231,10 @@ ghid_get_coords (const char *msg, int *x, int *y)
 gboolean
 ghid_note_event_location (GdkEventButton * ev)
 {
+  /* FIXME: IFDEF HACK */
+#ifdef ENABLE_GL
   extern float global_depth; /* FIXME: REMOVE THIS HACK! */
+#endif
   gint x, y;
   gboolean moved;
